### ISSUE TITLE
Pytorch: add more components for standard neural network modules

### ISSUE THREAD

**Issue by uakfdotb** [skyhookml/skyhookml#11]
VGG, ResNet, YOLOv4, Mask R-CNN, etc.

**Comment by uakfdotb** [comment]
* Object detection: YOLOv5, and add node for SSD. Also detectron2 as a separate node from pytorch. (SSD also needs a custom repository since we fixed something to make it work for inference.)
* Image classification: ResNet, VGG
